packages/action-password-update/src/components/PasswordUpdateDialog/PasswordUpdateDialog.scss

```diff
@@ -1,54 +1,206 @@
-.PasswordUpdateDialog {
+/*****
+* import sass dependencies
+*/
 
-  .oldPassword {
-    margin-bottom: 1.5em;
-  }
+// TODO WEB reduce redundancy
+// TODO WEB get rid of cumbersome imports
+// TODO WEB make it possible to use _variables.scss from customer
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_opacity.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_vendor-prefixes.scss";
+@import "../../../../../node_modules/bootstrap-sass/assets/stylesheets/bootstrap/mixins/_background-variant.scss";
+@import "../../../../../node_modules/font-awesome/scss/_variables.scss";
 
-  .ValidationRules {
-    margin-bottom: 1.5em;
-  }
+$fa-font-path: "../../../../../node_modules/font-awesome/fonts/";
+@import "../../../../../node_modules/font-awesome/scss/font-awesome.scss";
+
+
+/*****
+* tocco-client boilerplate
+* for styling convenience while developing, only used inside monorepo tocco-client
+*/
+
+// TODO CLIENT replace #root with .tocco-client-development for boilerplate styling
+#root {
+  margin: 50px;
+  border: 1px dashed $gray-lighter;
+}
```

Comment:
This rule should be removed. There is a index.html file with `<div id="root"></div>` (which is used in development environment only. You may style this root div right there in index.html if you want to (other pull request).